packages/chargebee/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "1.5.1",
+  "version": "1.6.0",
```

Comment:
you can get rid of the proptypes devDependency since it is already imported via the root package.json